<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="statusText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>641</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectfileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Media File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="androidHDRaioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Android HD</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="androidQHDRaioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Android qHD</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="appleHDRadioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Apple HD</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="appleFullHDRadioButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Apple Full HD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
